### ISSUE TITLE
Error when using UMD

### ISSUE THREAD

**Issue by kabanoki** [kevinfaguiar/vue-twemoji-picker#35]
Calling the following CDN usage to HTML will generate an error.

[https://cdn.jsdelivr.net/npm/@kevinfaguiar/vue-twemoji-picker@5.2.1/dist/vue-twemoji-picker.umd.js](https://cdn.jsdelivr.net/npm/@kevinfaguiar/vue-twemoji-picker@5.2.1/dist/vue-twemoji-picker.umd.js)

```
vue-twemoji-picker.umd.js:1666 
Uncaught TypeError: Cannot use 'in' operator to search for 'default' in undefined
```

How should I respond?

**Comment by kevinfaguiar** [comment]
Hi there! 
What browser are you using? (Also, version details, etc.)
Any specific environment info I should know to reproduce?

**Comment by kabanoki** [comment]
Thank you for your reply!
The environment I confirmed is as follows.

```
[OS]: Windows 10
[Chrome]: 80.0.3987.122
[Firefox]: 73.0.1 (64 bit)
```

**Demo**
[https://codepen.io/kabanoki/pen/mdJWrzp?editors=1011](https://codepen.io/kabanoki/pen/mdJWrzp?editors=1011)

**Souce Code**
[https://github.com/kabanoki/vue/tree/master/vue-twemoji-picker](https://github.com/kabanoki/vue/tree/master/vue-twemoji-picker)

**Comment by kevinfaguiar** [comment]
Hi there @kabanoki! 
There were some bugs in the UMD/IIFE builds but they have now been fixed beginning with version 5.4.0. There is documentation for using these builds available here: https://kevinfaguiar.github.io/vue-twemoji-picker/docs/getting-started/#using-umd-iife-builds
